build.gradle

```diff
@@ -59,7 +59,7 @@ dependencies {
     testImplementation (group: 'io.github.bonigarcia', name: 'webdrivermanager', version: '5.5.3') {
         exclude group: 'org.seleniumhq.selenium'
     }
-    testImplementation platform(group: 'org.seleniumhq.selenium', name: 'selenium-bom', version: '4.13.0')
+    testImplementation platform(group: 'org.seleniumhq.selenium', name: 'selenium-bom', version: '4.14.1')
```

Comment:
why we cannot use ${seleniumVersion} template there?

Reply:
For now it doesn't matter which form of version to put, because we stick to the latest version (`4.14.1`) everywhere. But once new version of Selenium is released, @dependabot will pick it up and open PR with update of the version in test dependencies. In this way CI tests will be run against the latest Selenium client, if any compatibility issue appear, we'll detect it immediately.

Reply:
why this time there was no PR from dependabot?

Reply:
@depenabot is run by schedule (https://github.com/appium/java-client/blob/master/.github/dependabot.yml#L3-L8): once a week (on Monday) for this repo. Or it can be triggered manually.

Reply:
I assume triggering it manually would still save more time in comparison to a custom PR creation